Description: Avoid samba/popt.h, which is not present in newer Samba versions
Author: Jelmer Vernooij <jelmer@debian.org>
Status: Cherrypicked from upstream

=== modified file 'doc/examples/multiple_notif.c'
--- old/doc/examples/multiple_notif.c	2010-11-24 17:15:37 +0000
+++ new/doc/examples/multiple_notif.c	2012-03-15 17:04:10 +0000
@@ -39,7 +39,7 @@
 
 #include <libmapi/libmapi.h>
 #include <pthread.h> 
-#include <samba/popt.h>
+#include <popt.h>
 
 #define	DEFAULT_PROFDB	"%s/.openchange/profiles.ldb"
 

=== modified file 'mapiproxy/libmapistore/tests/mapistore_test.c'
--- old/mapiproxy/libmapistore/tests/mapistore_test.c	2012-02-20 14:56:25 +0000
+++ new/mapiproxy/libmapistore/tests/mapistore_test.c	2012-03-15 17:04:10 +0000
@@ -23,7 +23,7 @@
 #include "mapiproxy/libmapistore/mapistore_errors.h"
 #include <talloc.h>
 #include <core/ntstatus.h>
-#include <samba/popt.h>
+#include <popt.h>
 #include <param.h>
 #include <util/debug.h>
 

=== modified file 'testprogs/check_fasttransfer.c'
--- old/testprogs/check_fasttransfer.c	2012-02-24 13:23:22 +0000
+++ new/testprogs/check_fasttransfer.c	2012-03-15 17:04:10 +0000
@@ -23,7 +23,7 @@
 #include "mapiproxy/libmapistore/mapistore.h"
 #include "mapiproxy/libmapistore/mapistore_errors.h"
 
-#include <samba/popt.h>
+#include <popt.h>
 #include <ldb.h>
 #include <talloc.h>
 #include <inttypes.h>

=== modified file 'testprogs/test_asyncnotif.c'
--- old/testprogs/test_asyncnotif.c	2011-10-25 10:05:22 +0000
+++ new/testprogs/test_asyncnotif.c	2012-03-15 17:04:10 +0000
@@ -21,7 +21,7 @@
 
 #include "libmapi/libmapi.h"
 
-#include <samba/popt.h>
+#include <popt.h>
 #include <talloc.h>
 
 static void popt_openchange_version_callback(poptContext con,

=== modified file 'utils/backup/openchangemapidump.c'
--- old/utils/backup/openchangemapidump.c	2011-06-27 21:26:39 +0000
+++ new/utils/backup/openchangemapidump.c	2012-03-15 17:04:10 +0000
@@ -21,7 +21,7 @@
 */
 
 #include "libmapi/libmapi.h"
-#include <samba/popt.h>
+#include <popt.h>
 #include <param.h>
 
 #include "openchangebackup.h"

=== modified file 'utils/exchange2mbox.c'
--- old/utils/exchange2mbox.c	2012-02-27 13:43:33 +0000
+++ new/utils/exchange2mbox.c	2012-03-15 17:04:10 +0000
@@ -20,7 +20,7 @@
 */
 
 #include "libmapi/libmapi.h"
-#include <samba/popt.h>
+#include <popt.h>
 #include <ldb.h>
 
 #include <sys/types.h>

=== modified file 'utils/mapiprofile.c'
--- old/utils/mapiprofile.c	2012-02-27 14:46:02 +0000
+++ new/utils/mapiprofile.c	2012-03-15 17:04:10 +0000
@@ -20,7 +20,7 @@
 */
 
 #include "libmapi/libmapi.h"
-#include <samba/popt.h>
+#include <popt.h>
 #include <param.h>
 #include "openchange-tools.h"
 

=== modified file 'utils/mapitest/mapitest.c'
--- old/utils/mapitest/mapitest.c	2010-12-01 20:59:03 +0000
+++ new/utils/mapitest/mapitest.c	2012-03-15 17:04:10 +0000
@@ -22,7 +22,7 @@
 #include "utils/mapitest/mapitest.h"
 #include "utils/openchange-tools.h"
 
-#include <samba/popt.h>
+#include <popt.h>
 #include <param.h>
 
 #include "config.h"

=== modified file 'utils/openchangeclient.c'
--- old/utils/openchangeclient.c	2012-02-28 14:15:09 +0000
+++ new/utils/openchangeclient.c	2012-03-15 17:04:10 +0000
@@ -23,7 +23,7 @@
 #include "libmapi/libmapi_private.h"
 #include "libmapi/mapi_nameid.h"
 #include "libocpf/ocpf.h"
-#include <samba/popt.h>
+#include <popt.h>
 #include <param.h>
 
 #include "openchangeclient.h"

=== modified file 'utils/openchangepfadmin.c'
--- old/utils/openchangepfadmin.c	2011-06-27 21:26:39 +0000
+++ new/utils/openchangepfadmin.c	2012-03-15 17:04:10 +0000
@@ -21,7 +21,7 @@
 
 #include "libmapi/libmapi.h"
 #include "libmapiadmin/libmapiadmin.h"
-#include <samba/popt.h>
+#include <popt.h>
 #include <param.h>
 
 #include "openchangepfadmin.h"

